tests/unit/adapters/test_local_storage.py

```diff
@@ -0,0 +1,188 @@
+# pylint: disable=protected-access
```

Comment:
Here I would expect to find tests for local storage access only, meanwhile the tests here call generate_*_filter and read_parquet functions which is the heart of the data.py module (=domain code here).

Reply:
True, it is hard to test separately. local_storage module has too many responsibilities and I think it needs to be structured so that it is easy to test one thing at a time.

Reply:
Moving the read parquet to domain/data like in our other thread would make this test only touch the /domain modules at least 🤔 Or/and I could define some "test filters" as constants in a file. That should be pretty straight forward too 👍🏻

Reply:
Can we try that?

Reply:
I am on it 🏃🏻